integrations/webchat/examples/custom-element/client/react-animation/src/App.js

```diff
@@ -47,6 +48,8 @@ function viewChangeHandler(event, instance, stylesInitializedRef) {
       // Move the main window to the off-screen position and then un-hide it.
       instance.elements.getMainWindow().addClassName('StartOpenAnimation');
       instance.elements.getMainWindow().removeClassName('HideWebChat');
+      // Make the custom element visible.
+      customElement.classList.remove('WebChatContainer--hidden');
```

Comment:
In this case, the launcher is visible when you first launch the app and you can use it to open web chat but when you close the web chat, the launcher does not become visible again and you can't open web chat back up.